various-receipes/message-queue/docker/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+pubsub-emulator:
```

Comment:
How do you know when it's ready?

Reply:
When the port 8085 is ready, I'll add something to validate if is ready 👍

Reply:
Maybe we should include small wait-for util in our repository?